console/console-init/ui/src/modules/address-space/dialogs/CreateMessagingProject/MessagingProjectReview.tsx

```diff
@@ -8,34 +8,29 @@ import { Loading } from "use-patternfly";
 import { useQuery } from "@apollo/react-hooks";
 import { ADDRESS_SPACE_COMMAND_REVIEW_DETAIL } from "graphql-module/queries";
 import { AddressSpaceReview } from "modules/address-space/components";
+import { IMessagingProject } from "./CreateMessagingProject";
 
-export interface IReviewProps {
-  name?: string;
-  type?: string;
-  plan?: string;
-  namespace: string;
-  authenticationService: string;
+interface IMessagingProjectReviewProps {
+  projectDetail: IMessagingProject;
 }
 
-export const Review: React.FunctionComponent<IReviewProps> = ({
-  name,
-  type,
-  plan,
-  namespace,
-  authenticationService
+export const MessagingProjectReview: React.FunctionComponent<IMessagingProjectReviewProps> = ({
+  projectDetail
 }) => {
+  const { name, namespace, type, plan, authService } =
+    projectDetail && projectDetail;
   const { data, loading } = useQuery(ADDRESS_SPACE_COMMAND_REVIEW_DETAIL, {
     variables: {
       as: {
         metadata: {
-          name: name,
-          namespace: namespace
+          name: name && name,
```

Comment:
There is no benefit of adding && condition for same level. It will handle by above destructuring change.
name: name,
namespace: namespace